src/modules/ShortcutGuide/ShortcutGuide/shortcut_guide.cpp

```diff
@@ -261,3 +234,61 @@ void OverlayWindow::get_exe_path(HWND window, wchar_t* path)
         }
     }
 }
+
+ShortcutGuideSettings OverlayWindow::GetSettings() noexcept
+{
+    ShortcutGuideSettings s;
+    json::JsonObject properties;
+    try
+    {
+        PowerToysSettings::PowerToyValues settings =
```

Comment:
Since `settings` was the suggested name for the `ShortcutGuideSettings` instance, you can rename `PowerToysSettings::PowerToyValues settings` -> `PowerToysSettings::PowerToyValues settingsValues` or something like that.